gran_source_list.tsv

```diff
@@ -11,7 +11,11 @@ usgs-r/EflowStats	v4.0.2
 GLEON/GLMr	v3.1.1
 ldecicco-usgs/EGRETci	v0.0.8
 usgs-r/smwrBase	1.0.4
-usgs-r/smwrGraphs	1.0.7
+usgs-r/smwrGraphs	v1.0.10
 usgs-r/glmtools	v0.9.3
 usgs-r/powstreams	v0.5.1
+<<<<<<< HEAD
```

Comment:
OK, So the source was that my .tsv file had the old version number and when
I pulled from master, those lines were created to indicate the difference.
I fixed mine and created a new pull request.


On Mon, May 18, 2015 at 3:33 PM, Jordan Walker <notifications@github.com>
wrote:

> In gran_source_list.tsv
> <https://github.com/USGS-R/grantools/pull/50#discussion_r30544605>:
>
> >  usgs-r/glmtools	v0.9.3
> >  usgs-r/powstreams	v0.5.1
> > +<<<<<<< HEAD
>
> This commit is suggesting that the line "<<<<<<< HEAD" be added along with
> the other two that are part of the textual diff of the changes. These lines
> need to be removed, along with the line you intend to replace (line 20).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/USGS-R/grantools/pull/50/files#r30544605>.
>